### DISCUSSION TITLE
How to show comments on my blog?

### DISCUSSION THREAD

**Discussion by ohnonot** [datenstrom/community#55] [Ask a question] [answered]
The issue:
I would like to have comments on my website/blog.
All I see is a disqus plugin, but I don't want to use disqus (or any cloud service).
Is it possible to have an internal comment system - however simple - with yellowcms?
I have rudimentary knowledge of php so I could try to "port" something, if someone would give me a push in the right direction.

The pleasantries:
I love yellowcms! I've been trying out dozens of more-or-less minimal flat-file cms recently, and this one just feels right. The combination of bareboned, yet noob friendly. The fact that the admin backend is _very_ minimal and **optional**. I hope it stays that way. I prefer editing files, although I'm not a full-time coder. Thanks.

**Comment by markseuffert** [comment]
Thanks for your feedback!

Yes you're right, for user feedback there's [Disqus](https://github.com/GiovanniSalmeri/yellow-disqus) and [Contact](https://github.com/annaesvensson/yellow-contact). Maybe someone has a suggestion for a commenting system or wants to write one? Any help would be much appreciated.

**Comment by conchitawurst** [comment]
Here are two commenting systems which could be used:
https://github.com/monperrus/jskomment
https://github.com/jacobwb/hashover
As I got it right, both are flat-file, so both should be easily integratable without the need of special requirements. However, I have to admit that I'm unskilled in any programming. Therefore, I can't try integrating myself, but maybe someone else is so nice!

**Comment by wunderfeyd** [comment, marked as answer]
Please checkout the latest [yellow-comments](https://github.com/nasendackel/yellow-comments). It's ready for the most simple things (I hope). Your feedback there is greatly appreciated.

**Reply by annaesvensson** [reply]
This is now one of the most popular extensions, maybe time to remove the experimental status and publish it?

**Reply by annaesvensson** [reply]
I propose that Giovanni's [comment extensions](https://github.com/GiovanniSalmeri/yellow-comment) be made publicly available and no longer experimental. 😀

**Reply by GiovanniSalmeri** [reply]
Proposal gladly accepted! 😀 I will proceed with publication in the next few days.
